integration-tests/bats/sql-privs.bats

```diff
@@ -60,12 +60,12 @@ teardown() {
     start_sql_server_with_config test_db server.yaml
 
     server_query test_db 1 "select user from mysql.user order by user" "User\ndolt"
-    run ls -a
-    [[ "$output" =~ ".doltcfg" ]] || false
-
     server_query test_db 1 "create user new_user" ""
     server_query test_db 1 "select user from mysql.user order by user" "User\ndolt\nnew_user"
 
+    run ls -a
```

Comment:
This is for the file in general. I looks like you haven't added any new tests for the new functionality. We'll also need tests for the additions I mentioned previously.